aquadoggo/Cargo.toml

```diff
@@ -27,6 +27,7 @@ asynchronous-codec = { version = "0.6.2", features = ["cbor"] }
 axum = { version = "0.6.10", features = ["headers"] }
 bamboo-rs-core-ed25519-yasmf = "0.1.1"
 bs58 = "0.4.0"
+bytes = "1.4.0"
```

Comment:
We already have this dependency with `asynchronous_codec` but I'm still adding it explicitly here